### DISCUSSION TITLE
Gradle 8.1.1 support

### DISCUSSION THREAD

**Discussion by seksky333** [robolectric/robolectric#8443] [Q&A] [answered]
Our team is trying to upgrade to gradle 8.1.1 from gradle 8.0.2, and the robolectric version we have is 4.10.3. But our script - generateAuDebugUnitTestConfig is failing and an error says  "Unable to find manifest output". I am not sure if this is something to do with the compatibility with robolectric or not.

Please close this post if this question is not supposed to be asked here. 

Thanks in advance!

**Comment by utzcoz** [comment, marked as answer]
@seksky333 It looks like related to AGP more. IIRC, Robolectric doesn't resolve any manifest as AGP processes them together, and Robolectric just reads generated manifest file by AGP. I recommend you to file a new issue at https://issuetracker.google.com/issues.

**Reply by chestersek-kogan** [reply]
Thank you!
